### DISCUSSION TITLE
Can conductor be registered to the registry such as Zookeeper、Console or Alibaba NACOS?

### DISCUSSION THREAD

**Discussion by wpz850398226** [Netflix/conductor#2797] [Q&A]
Hello, I'm having some trouble,please help me, thank you
    The main one introduction about Conductor I saw on Github was "Conductor is a microservices orchestration engine", as far as I know,microservices are all need to be registered to the registry,Calls between microservices are called by service names rather than directly from IP addresses.
    Can conductor service be registered to the registry such as Zookeeper、Console or Alibaba NACOS?If so, how to make call to other microservices by service name in a workflow?Use HTTP task? I don't think HTTP task will be useful.

**Comment by aravindanr** [comment]
Conductor is extensible and support user driven tasks. Conductor administrators and users can create their own tasks. In your case, i believe, a task can be created to call a service using its name.

**Reply by wpz850398226** [reply]
What do you mean by "user driven tasks"?Does it mean SIMPLE task?

**Reply by wpz850398226** [reply]
Using SIMPLE task or other user driven tasks requires business code  to follow certain coding rules,that leads to code intrusion;
What I want is to call other microservices without any others' code structure modification,just like HTTP task,but works in LAN and by service name;

**Reply by wpz850398226** [reply]
![image](https://user-images.githubusercontent.com/41596989/156129857-a41b811a-bc40-4b92-a9ae-08de74630f65.png)
Does it means user-defined task types here?
If so,that's what I want!
I want to define a new task type to call other microservice by service name,and I will implement the business logic myself;

**Reply by aravindanr** [reply]
> Conductor administrators and users can create their own tasks.

I am assuming that your role is Conductor administrator, someone who manages Conductor for other users in your company. In order for you to define a new task type that your users can use, System Tasks is the way to go. System tasks can be synchronous (JSON_JQ_TRANSFORM) or asynchronous (HTTP). Synchronous tasks block the workflow evaluation thread until the task execution completes. Asynchronous tasks are put into a queue and evaluated on a different thread. For your particular use-case, asynchronous seems best. 

Let us know if you need help configuring a system task.
